syne_tune/blackbox_repository/conversion_scripts/scripts/fcnet_import.py

```diff
@@ -23,7 +23,11 @@
 import pandas as pd
 import numpy as np
 import ast
-import h5py
+
+try:
+    import h5py
+except ImportError:
+    print("Cannot import h5py. Use 'pip install h5py'")
```

Comment:
Rather than having custom messages, it would be better to just use print the stack trace.
```
import traceback
try:
  import h5py
except ImportError:
  traceback.print_exc()
```
This way we dont need to write custom messages for everything but most importantly, when error happens users see the stack trace and can see what triggered the issue (maybe it was a file that they did not want to import for instance).

Reply:
The problem is that these messages are also printed when h5py is not used, which is totally fine. That is the price for having the import on top of the file.

Reply:
This recipe will be only imported if you want to get access to blackboxes so it may a problem then, that being said I am fine leaving a custom comment there if you prefer this option.